javascript-carousel/index.html

```diff
@@ -0,0 +1,100 @@
+          <i class="far fa-circle circle-btn circle-btn-five"></i>
+        </div>
+      </div>
+      <i class="fas fa-chevron-right arrow-button arrow-btn-right"></i>
+    </div>
+  </div>
+
+  <div class="view two hidden">
+    <div class="img-container">
+      <i class="fas fa-chevron-left arrow-button arrow-btn-left"></i>
+      <div>
+        <img src="images/004.png" alt="Charmander">
+        <div class="circle">
+          <i class="far fa-circle circle-btn circle-btn-one"></i>
+          <i class="fas fa-circle circle-btn circle-btn-two"></i>
+          <i class="far fa-circle circle-btn circle-btn-three"></i>
+          <i class="far fa-circle circle-btn circle-btn-four"></i>
+          <i class="far fa-circle circle-btn circle-btn-five"></i>
+        </div>
+      </div>
+     <i class="fas fa-chevron-right arrow-button arrow-btn-right"></i>
+    </div>
+  </div>
+
+  <div class="view three hidden">
+    <div class="img-container">
+      <i class="fas fa-chevron-left arrow-button arrow-btn-left"></i>
+      <div>
+        <img src="images/007.png" alt="Squirtle">
+        <div class="circle">
+          <i class="far fa-circle circle-btn circle-btn-one"></i>
+          <i class="far fa-circle circle-btn circle-btn-two"></i>
+          <i class="fas fa-circle circle-btn circle-btn-three"></i>
+          <i class="far fa-circle circle-btn circle-btn-four"></i>
+          <i class="far fa-circle circle-btn circle-btn-five"></i>
+        </div>
+      </div>
+      <i class="fas fa-chevron-right arrow-button arrow-btn-right"></i>
+    </div>
+  </div>
+
+  <div class="view four hidden">
+    <div class="img-container">
+      <i class="fas fa-chevron-left arrow-button arrow-btn-left"></i>
+      <div>
+        <img src="images/025.png" alt="Pikachu">
+        <div class="circle">
+          <i class="far fa-circle circle-btn circle-btn-one"></i>
+          <i class="far fa-circle circle-btn circle-btn-two"></i>
+          <i class="far fa-circle circle-btn circle-btn-three"></i>
+          <i class="fas fa-circle circle-btn circle-btn-four"></i>
+          <i class="far fa-circle circle-btn circle-btn-five"></i>
+        </div>
+      </div>
+      <i class="fas fa-chevron-right arrow-button arrow-btn-right"></i>
+    </div>
+  </div>
+
+  <div class="view five hidden">
+    <div class="img-container">
+      <i class="fas fa-chevron-left arrow-button arrow-btn-left"></i>
+      <div>
+        <img src="images/039.png" alt="Jigglypuff">
+        <div class="circle">
+          <i class="far fa-circle circle-btn circle-btn-one"></i>
+          <i class="far fa-circle circle-btn circle-btn-two"></i>
+          <i class="far fa-circle circle-btn circle-btn-three"></i>
+          <i class="far fa-circle circle-btn circle-btn-four"></i>
+          <i class="fas fa-circle circle-btn circle-btn-five"></i>
+        </div>
+      </div>
+      <i class="fas fa-chevron-right arrow-button arrow-btn-right"></i>
+    </div>
```

Comment:
You dont want 5 carousels that you switch between, you want one carousel with 5 images you switch between.

Reply:
Oh got it! Okay I'll make that change.

Reply:
New gif for the updated version:

![20201210145519478](https://user-images.githubusercontent.com/69181727/101839933-198e7900-3af8-11eb-965c-18e3d87f0fa4.gif)